web/controllers/notes_controller.ex

```diff
@@ -2,6 +2,13 @@ defmodule Notox.NoteController do
   use Notox.Web, :controller
   alias Notox.Note
 
+  def index(conn, %{"search" => %{"query" => query}}) do
+    notes =
+      Note.search(Note, query)
```

Comment:
Ok that's interesting. I've faced this decision a few times. Is your general rule to avoid a pipeline if there's only one, but use the pipeline for everything if things are already being piped? For instance:

```
# Bad
Note |> Note.search(query)
 # Good
Note.search(Note, query)

# Bad
Note.search(Note, query)
|> Repo.all

# Good
Note
|> Note.search(query)
|> Repo.all
```